website/client/js/services/userServices.js

```diff
@@ -512,15 +512,15 @@ angular.module('habitrpg')
         },
 
         addWebhook: function (data) {
-          callOpsFunctionAndRequest('addWebhook', 'webhook', "POST", '', data, data.body);
```

Comment:
I think data.body is correct here as `{body: {}}` will be passed as `data` and we only want to send data.body to the server